tests/test_latent_diffusion.py

```diff
@@ -250,25 +254,29 @@ def test_train():
          clean_images = clean_images.to(device)
          class_labels = class_labels.to(device)
 
+         # Encode image with VAE
+         with torch.no_grad():
+            clean_images = vae.encode(clean_images.to(device)).latent_dist.sample()
+
          # Sample noise to add to the images
          noise = torch.randn(clean_images.shape).to(clean_images.device)
 
          # Sample a random timestep for each image
          timesteps = torch.randint(
                0,
-               image_pipe.scheduler.num_train_timesteps,
+               scheduler.num_train_timesteps,
                (batch_size,),
                device=device,
          ).long()
 
          # Forward diffusion process (Add noise to the clean images according to the noise magnitude at each timestep)
-         noisy_images = image_pipe.scheduler.add_noise(clean_images, noise, timesteps)
+         noisy_images = scheduler.add_noise(clean_images, noise, timesteps)
 
          # Get the model prediction for the noise
          if add_conditioning:
-               noise_pred = image_pipe.unet(noisy_images.float(), timesteps, class_labels=class_labels, return_dict=False)[0]
+               noise_pred = model(noisy_images.float(), timesteps, class_labels=None, return_dict=False)[0]
```

Comment:
Should this line include `class_labels=class_labels`?  If it was removed for debugging purposes that makes sense.